exonum/tests/websocket/main.rs

```diff
@@ -427,3 +427,53 @@ fn test_node_shutdown_with_active_ws_client_should_not_wait_for_timeout() {
         let _ = client.shutdown();
     }
 }
+
+#[test]
+fn test_sending_message_size() {
+    let max_message_len = 512_usize;
+    let node_handler = run_node_with_message_len(6338, 8087, max_message_len as u32);
+    let mut client =
+        create_ws_client("ws://localhost:8087/api/explorer/v1/ws").expect("Cannot connect to node");
+    client
+        .stream_ref()
+        .set_read_timeout(Some(Duration::from_secs(10)))
+        .unwrap();
+
+    // Send transaction.
+    let (pk, sk) = gen_keypair();
+    let name = String::from_utf8(vec![64; max_message_len / 2]).unwrap();
```

Comment:
Please use the length here (or as a separate successful case) so that

```rust
assert_eq!(tx.signed_message().raw().len(), max_message_len);
```

I suppose that the best way to find the correct value is by using binary search, but it's possible to hard-code it (the value is 371).

Also, it's possible to use `"A".repeat(x)` instead of `String::from_utf8`.